webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -5483,23 +5483,23 @@ var MCK_CHAT_POPUP_TEMPLATE_TIMER;
       			_this.getContactImageLink = function(contact, displayName, message) {
                         var imgsrctag = '';
                         var profileDisplayName = displayName ? displayName + ' profile image' : 'Profile image';
-                        if(!contact.isGroup){
+                        if(contact && !contact.isGroup){
```

Comment:
@CaptainT33mo is contact && check required here? i guess if it is not coming, nothing will work in code below it, we can probably figure out why contact is coming as undefined and fix that part.

Reply:
@devashishmamgain actually I am sending `null` in that function like this: `_this.getContactImageLink(null, name)`
So, in that case, it'll throw error `Cannot read property isGroup of undefined`.